recipes-core/initrdscripts/files/ni_provisioning.safemode

```diff
@@ -181,13 +181,15 @@ has_tpm_chip ()
 	[ -e "/dev/tpm0" ]
 }
 
-check_for_tpm_chip()
+check_if_ramfs_required()
 {
 	# If use_ramfs was unspecified by user, set it based on
-	#  presence of TPM chip
+	#  presence of TPM chip or Hyper-V
 	if [ -z "$use_ramfs" ]; then
 		if has_tpm_chip; then
 			use_ramfs=true
+		elif lsmod | grep hv_storvsc &> /dev/null; then
```

Comment:
What loads the hv_storvsc driver in the provisioning disk?  Does the provisioning disk always use an initramfs?

Reply:
I added the drivers in a [previous commit](https://github.com/ni/meta-nilrt/commit/bc80fdde2156ba33641eaf740e762d1c00c03eda) to load the drivers. Yes, both the provisioning mode and safemode always use initramfs.